nodeJS/express/installing_postgresql.md

```diff
@@ -0,0 +1,293 @@
+<!-- markdownlint-disable MD024 TOP004 -->
+
+### Introduction
+
+Data persistence is integral to the vast majority of web applications. Data persistence is achieved through databases. Being able to understand how to structure, build, and query your own database are important skills for any full-stack developer to have.
+
+For example, consider The Odin Project. Where is your username stored? Where are project submissions stored? Where does this lesson content, the one you're reading right now, get stored? Yes, a database!
+
+We've chosen [PostgreSQL](https://www.postgresql.org/) as our database of choice for the curriculum. All SQL knowledge that you've learned in the earlier course should be transferable to PostgreSQL. We'll also learn new PostgreSQL specific things. Watch through this [fun and short introduction video to PostgreSQL by Fireship](https://www.youtube.com/watch?v=n2Fluyr3lbc).
+
+<div class="lesson-note lesson-note--critical" markdown="1">
+
+#### Prerequisite SQL knowledge
```

Comment:
Is this note box necessary? If this lesson is placed after the SQL course it seems unnecessary to tell users that they should have done some other course first.

If it ultimately is necessary, I'd opt to removing this header and instead just have the content of the note box speak for itself. I don't think we would really *need* a separate heading for this.

Reply:
I think it's necessary. When we release the new updates, I'd expect _some_ people to jump straight into the new lessons in the Node course without going through the newly added SQL course. 

Sure, I'll remove the headings.